homeassistant/components/onewire/sensor.py

```diff
@@ -100,19 +194,26 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 class OneWire(Entity):
     """Implementation of an One wire Sensor."""
 
-    def __init__(self, name, device_file, sensor_type):
+    def __init__(self, name, device_file, sensor_type, owproxy=None):
```

Comment:
Implement an init method in the child class instead that accepts all arguments as positional arguments and pass the common arguments to the parent class via `super().__init__(...)`.